### ISSUE TITLE
Update Macedonian translations

### ISSUE THREAD

**Issue by LordDeatHunter** [squidfunk/mkdocs-material#6511]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "mk",
  "action.edit": "Уредете ја оваа страница",
  "action.skip": "Прескокнете до содржината",
  "action.view": "Прикажи го изворот на оваа страница",
  "announce.dismiss": "Не го прикажувај ова повторно",
  "blog.archive": "Архива",
  "blog.categories": "Категории",
  "blog.categories.in": "во",
  "blog.continue": "Продолжи со читање",
  "blog.draft": "Нацрт",
  "blog.index": "Назад кон индекс",
  "blog.meta": "Метадата",
  "blog.references": "Поврзани линкови",
  "clipboard.copy": "Копирај во таблата",
  "clipboard.copied": "Копирано",
  "consent.accept": "Прифати",
  "consent.manage": "Прилагодување со поставките",
  "consent.reject": "Одбиј",
  "footer": "Подножје",
  "footer.next": "Следно",
  "footer.previous": "Претходно",
  "header": "Заглавје",
  "meta.comments": "Коментари",
  "meta.source": "Извор",
  "nav": "Наслов за навигација",
  "readtime.one": "1 min читање",
  "readtime.other": "# min читање",
  "rss.created": "RSS извор",
  "rss.updated": "RSS извор на ажурирана содржина",
  "search": "Пребарување",
  "search.placeholder": "Пребарување",
  "search.share": "Сподели",
  "search.reset": "Чисти",
  "search.result.initializer": "Иницијализирање на пребарувањето",
  "search.result.placeholder": "Напишете за да започнете со пребарување",
  "search.result.none": "Нема соодветни документи",
  "search.result.one": "1 документ што се совпаѓа",
  "search.result.other": "# соодветни документи",
  "search.result.more.one": "Уште 1 на оваа страница",
  "search.result.more.other": "Уште # на оваа страница",
  "search.result.term.missing": "Недостасува",
  "select.language": "Изберете јазик",
  "select.version": "Изберете верзија",
  "source": "Одете до складиштето",
  "source.file.contributors": "Соработници",
  "source.file.date.created": "Создаден",
  "source.file.date.updated": "Последно ажурирање",
  "tabs": "Јазичиња",
  "toc": "Содржина",
  "top": "Вратете се на почетокот"
}[key] }}{% endmacro %}
```

### Country flag

:flag_mk:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [X] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in e8745daa6.

**Comment by squidfunk** [comment]
Released as part of [9.5.3](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.3).
